modeltranslation/static/modeltranslation/js/tabbed_translation_fields.js

```diff
@@ -440,7 +440,7 @@ var google, django, gettext;
 
     var MainSwitch = {
       languages: [],
-      $select: $("<select>"),
+      $select: $("<select class='modeltranslation-main-switch'>"),
```

Comment:
```suggestion
      $select: $("<select id='modeltranslation-main-switch' class='modeltranslation-switch'>"),
```

@hovi why do you think class is better?

It's single main switch. Maybe we should add id and class? :)

Reply:
I understand, that from your point of view it is something unique, that is currently only once on the page. 
From css code point of view it gives no benefit to use idinstead of id.
I am happy either way, having both class and id is good compromise. I will modify PR. Should ID and class have same value?

Reply:
I think they should be different, like in my suggestion. You can just click "Commit suggestion".